vmm/sandbox/src/container/handler/spec.rs

```diff
@@ -71,6 +77,20 @@ where
         }
         // Update sandbox files mounts for container
         container_mounts(&shared_path, spec);
+
+        // Update ipc and uts namespace for container
+        if let Some(linux) = spec.linux.as_mut() {
```

Comment:
there is a namespace handler, which is doing similar things

Reply:
OK